exonum/tests/balance_service.rs

```diff
@@ -159,7 +180,9 @@ pub mod service {
         api_tx.send(tx_copy).unwrap();
```

Comment:
This test was about double tx execution, probably cloned tx was executed at second block?
But now test wait only one block.

May be we should write test as follow:
```
api_tx.send(tx_orig);
receiver.wait_for_message();
api_tx.send(tx_orig);
receiver.wait_for_message();
```

Additionally just after `api_tx.send`, but before tx was executed, there possible state where `height > 0`.